[skip ci] Fix overlapping text in the guide

arye-eidelman opened the pull request:
### Summary

Added 21em of margin-right to the main content when the index is covering the right side

Fix #33406

twitnithegirl reviewed (commented):
I pulled this down and took a look and it fixes the overlapping text problem and looks good. I checked it in Chrome and Safari.

Just FYI @arye-dov-eidelman  if you ever have a PR that is just a change in the guides, you should include [skip ci] in the commit message so that the PR won't hog CI jobs.

twitnithegirl commented on guides/assets/stylesheets/main.css:
```diff
@@ -280,6 +280,10 @@ body {
+  #container .wrapper { padding-right: 21em; }
+}
+
 /* Links
```
Here, if you move your media query above the one on line 279 you can drop the `and (min-width: 800)` since it would be implied due to cascading. You'd have to reset the `#container .wrapper {padding-right: 0}` inside the `max-width:800` media query but  using the cascade instead of declaring the min-width seems to be more in line with how media queries are being used in the rest of the file.

twitnithegirl reviewed (approved):
This refactor is 💯

twitnithegirl commented:
Bumping this because while it isn't a super high priority change but I think it should get merged in. It fixes a pretty annoying visual bug in the docs. @kamipo

arye-eidelman committed:
[skip ci] Fix overlapping text in the guide

Fix a layout issue in the rails guides, where the navigation covers the main text, if the page is between 800 and 960 pixels wide.

Issue #33406

arye-eidelman committed:
[skip ci] Fix overlapping text in the guide

Fix a layout issue in the rails guides, where the navigation covers the main text, if the page is between 800 and 960 pixels wide.

Fix Issue #33406

arye-dov-eidelman commented:
I messed up my repository so I created a new pull request https://github.com/rails/rails/pull/33645

kamipo commented:
Closing in favor of #33645.
